libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationConstants.java

```diff
@@ -57,6 +57,11 @@ private NavigationConstants() {
    */
   public static final long ALERT_VIEW_PROBLEM_DURATION = 10000;
 
+  /**
+   * Mapbox shared preferences file name
+   */
+  public static final String MAPBOX_SHARED_PREFERENCES = "MapboxSharedPreferences";
```

Comment:
Yes for now, because this class hasn't been refactored yet and is outside `internal` package. So I cannot access it from within `internal`

Reply:
Ohh, yeah wondering if we should put it under the class in which is used. IMO, constants, objects and methods should live / be where their context is. From my experience utility and constants classes become a drawer easily and we should treat them carefully.

Reply:
This is the only constant I put in `NavigationConstants` because if in the future we use more SharedPreferences, we would want to make sure that we use the same file name. Putting the SharedPreference in the same file name makes it error prone in that different files might give different names to SharedPreference

Reply:
What about make it internal accessible from let's say `MapboxNavigationAccounts`? That way there'll be one single-entry - removing the possibility of giving it different names.